### DISCUSSION TITLE
Can the legacy toolbar color be changed?

### DISCUSSION THREAD

**Discussion by TrantaLocked** [Maplespe/DWMBlurGlass#317] [Q&A]
Can the color of the toolbar under the title bar be changed, both background and text? It's always the same blue/grey mica color when it is visible in apps including Task Manager, Notepad and others that allow the Aero theme title bar like Adobe Premiere.

![329104438-129f3660-843a-4b03-b8f1-30e87debd0d1](https://github.com/Maplespe/DWMBlurGlass/assets/18237916/aa88d6a2-f2a8-4827-a6f5-a7fb3278523a)

**Comment by 7kt4** [comment]
It's an msstyles thing

**Reply by TrantaLocked** [reply]
The one thing I can't figure out is how to change the separator line color between the Menu bar and the content from the default white. I tried adding various properties in the SEPARATOR part but to no avail.

Progress: https://github.com/nptr/msstyleEditor/discussions/140#discussioncomment-9394113
